### PR TITLE
DR Strategy Update - 2H 2023

### PR DESCRIPTION
**Pull request by batemansogq**
Before selecting the "Create pull request" button:  

1. Enter a meaningful title above^, using a prefix if necessary and keywords "New" or "Update" indicating the nature of changes.

2. Describe the summary, scope, and intent of this PR:  
Bring the DR strategy up to date
- various hyperlink updates
- updating Cognitive Services and Search to their new names AI Services and Search
- updating Chaos studio section, due to product uplift
- removed specific date references in the Synapse sections, due to PG focus on Fabric

3. Insert links(s) to any related work item(s) or supporting detail:  
[DELETE-OR-REPLACE-THIS-TEXT]   

  

  
<details><summary>AFTER YOUR PR HAS BEEN CREATED, expand this section for tips and additional instructions.</summary>    
      

These are common guidelines for contributions across the repos managed by the Cloud Architecture Content Team (CACT). Some repositories may have additional specific requirements that are not listed here.   

## Guidance for all contributors  
  
  | **Topic** | **Guidance** |
  | ----------| ------------ |
  | **Draft PR** | If your PR will be a work-in-progress for more than a day or two, select the **Convert to draft** link in the upper right of the page (under **Reviewers**) to change it to a draft. For future reference, you can also do this using the **Create pull request** button drop-down during PR creation. | 
  | **ms.date metadata** | <ul><li>Don't update an article's "ms.date" metadata property unless you've done a **full freshness review** of the content. A full freshness review includes changes required to correct or improve the **full** technical accuracy of the article.</li><li>Don't update "ms.date" if you're doing targeted changes to improve non-technical aspects of the article, such as editorial quality, art improvements, article template alignment, etc.</li><li>If you've changed any "ms.date" properties for work that wasn't part of full review for freshness, please reset them to their previous value.</li></ul> | 
  | **Placement and linking** | If you're creating a new article or articles, include updates to the related TOC.yml file to propose where the article(s) should be placed. Also consider other places within the document set where it would be beneficial to cross-reference and link to your new article(s). | 
  | **PR build** | After you open your PR, and for each successive commit that you push to your branch, the publishing platform will run validation on the files in your pull request. A summary of the build results for each file will be inserted inline into your pull request, which includes any build suggestions/warnings/errors. PRs cannot be merged until all build errors and most warnings are resolved. |
  | **Publishing** | Following a successful merge, most repos publish to the live site at least once per (business) day, usually around 10am Pacific. |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li></ul>
  
## Additional guidance for private repos and internal contributors  

  | **Topic** | **Guidance** | 
  | ----------| ------------ | 
  | **PR size** | If your PR is more than ~5 lines of changes, or you'd like for the changes to go through editorial or larger review, open a contribution request at https://aka.ms/Contribution and include a link to the PR in response #8. Once it's processed, you'll be notified of the next steps.  |
  | **PR title prefix** | Select the **Edit** button to the right of the PR title if you need to revise it. The following prefixes are reserved for specific contribution types:<br/><br/><ul><li>**[Quality Check]** - maintenance work related to content quality (edit passes, art improvements, template alignment)</li><li>**[LinkFix]** - recurring/adhoc PRs to correct link URLs</li><li>**[Pipeline]** - new/updated contributor success pipeline content</li><li>**[WIP]** - a work-in-progress draft requiring several days/weeks</li></ul> |
  | **PR preview** | Following successful build of your PR, publishable files will also include **Preview URL** links to staged previews of your new/updated articles. Be sure to review these for verification of your intended contributions, or to send to other internal contributors for review. |
  | **PR sign-off (public repo)** | If an article you own is updated in a public repo PR, you are responsible for sign-off. You will be automatically notified via email. The PR will not be merged until you've had a chance to review and sign-off. |
  | **PR sign-off (private repo)** | After you've completed your proposed changes, addressed build warnings, and completed all review work, you can begin the sign-off process for review and merge:<br/><br/><ol><li>If your PR is in draft mode, remove "[WIP]" from the title and select **Ready for review** button at the bottom of the PR.</li><li>Enter "#sign-off" in a new comment. This comment indicates that **you're confident the work meets or exceeds Microsoft's standards for publication**, and will trigger the review process.</li><li>Your PR may be selected for initial review by the CACT. Following CACT review, you may receive questions or requests for additional changes. You should have initial feedback from CACT review within a few business days. If you have an urgent request or need to contact the team, please mention `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers` in your PR and someone will get back to you. After CACT review is complete, a `CACT #sign-off` will be added.</li><li>Final review/merge is done by the PR review team. The PR team may also respond with feedback, categorized as "Blocking" (requires action from you), or "Non-blocking" (to be addressed in a future PR).</li></ol> |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com internal contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li><li>Authoring templates: [architecture-center-pr](https://review.learn.microsoft.com/help/contribute/architecture-center/templates/sample-solution-templates?branch=main), [well-architected-pr](https://review.learn.microsoft.com/help/contribute/global-waf-template?branch=main)</li><li>To contact the CACT use [e-mail](mailto:cact-pr-reviewers@microsoft.com?subject=Help%20with%20pull%20request), or @mention our GitHub team in your PR comments using: `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers`</li></ul>
</details>

### CODE REVIEW

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-recommendations-content.md [hunk 1]
```diff
@@ -22,9 +22,9 @@
...
-    1. Define what an outage is and how it maps to Microsoft’s definition of an outage. The Microsoft definition is available on the [Azure Service Level Agreement](https://azure.microsoft.com/en-us/support/legal/sla/) page at the product or service level
+    1. Define what an outage is and how it maps to Microsoft’s definition of an outage. The Microsoft definition is available on the [Azure Service Level Agreement](/azure.microsoft.com/support/legal/sla/) page at the product or service level
     1. An efficient monitoring and alerting system with accountable teams to review those metrics and alerts in a timely manner will help meet the goal
-1. Composite SLAs mean that the more components you have in your architecture, the higher the probability of a failure. You could use composite SLA to define an outage
+1. [Composite SLAs](/azure/well-architected/reliability/metrics#:~:text=After%20you%20gather%20the%20SLAs%20for%20the%20individual%20workload%20components%2C%20calculate%20a%20composite%20SLA) mean that the more components you have in your architecture, the higher the probability of a failure. You could use composite SLA to define an outage
```
Comment (ckittel) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-recommendations-content.md:27]:
```suggestion
1. [Composite SLAs](/azure/well-architected/reliability/metrics) mean that the more components you have in your architecture, the higher the probability of a failure. You could use composite SLA to define the chances of a component outage.
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-recommendations-content.md [hunk 2]
```diff
@@ -22,9 +22,9 @@
     1. Automation/job scheduler
     1. Source repository and deployment pipelines (GitHub, Azure DevOps)
 1. Early detection of outages is also a way to decrease RTO and RPO values significantly. Here are a few aspects that should be covered:
-    1. Define what an outage is and how it maps to Microsoft’s definition of an outage. The Microsoft definition is available on the [Azure Service Level Agreement](https://azure.microsoft.com/en-us/support/legal/sla/) page at the product or service level
+    1. Define what an outage is and how it maps to Microsoft’s definition of an outage. The Microsoft definition is available on the [Azure Service Level Agreement](/azure.microsoft.com/support/legal/sla/) page at the product or service level
```
Comment (ckittel) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-recommendations-content.md:25]:
```suggestion
    1. Define what an outage is and how it maps to Microsoft’s definition of an outage. The Microsoft definition is available on the [Azure Service Level Agreement](https://azure.microsoft.com/support/legal/sla/) page at the product or service level.
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-deploy-this-scenario-content.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ For Azure services
...
-    - If there are ever issues accessing the Status page, go to @AzureSupport on Twitter
+    - If there are ever issues accessing the Status page, go to @AzureSupport on X (formerly Twitter)
 - If impact/issues don’t match the incident (or persist after mitigation), then [contact support](https://www.aka.ms/AzurePortalSupportRequest) to raise a service support ticket
 
 For Power BI
 
 - The [Service Health](https://portal.office.com/AdminPortal/Home) page within their Microsoft 365 admin center will provide the latest updates
     - If there are issues accessing Service Health, refer to the [Microsoft 365 status page](https://status.office.com)
-    - If impact/issues don’t match the incident (or if issues persist after mitigation), they should contact support to raise a [service support ticket](https://dynamics.microsoft.com/en-us/support/)
+    - If impact/issues don’t match the incident (or if issues persist after mitigation), they should contact support to raise a [service support ticket](https://learn.microsoft.com/power-bi/support/create-support-ticket)
```
Comment (ckittel) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-deploy-this-scenario-content.md:28]:
```suggestion
    - If impact/issues don't match the incident (or if issues persist after mitigation), you should raise a [service support ticket](/power-bi/support/create-support-ticket).
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 4]
```diff
@@ -273,15 +273,15 @@ The following tables present a breakdown of each Azure service and component use
...
     - Notes
-        - As of Mar 2023, Azure Synapse Analytics only supports disaster recovery for [dedicated SQL pools](/azure/synapse-analytics/sql-data-warehouse/backup-and-restore?source=recommendations#geo-backups-and-disaster-recovery) and [doesn’t support it for Apache Spark pools](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-defender-for-key-vault-deploy-to-azure-synapse/ba-p/3201308#:~:text=Azure%20Synapse%20Analytics%20only%20supports%20disaster%20recovery%20for,snapshots%20for%20disaster%20recovery%20of%20dedicated%20SQL%20pools.)
+        - Currently, Azure Synapse Analytics only supports disaster recovery for [dedicated SQL pools](/azure/synapse-analytics/sql-data-warehouse/backup-and-restore?source=recommendations#geo-backups-and-disaster-recovery) and [doesn’t support it for Apache Spark pools](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-defender-for-key-vault-deploy-to-azure-synapse/ba-p/3201308#:~:text=Azure%20Synapse%20Analytics%20only%20supports%20disaster%20recovery%20for,snapshots%20for%20disaster%20recovery%20of%20dedicated%20SQL%20pools.)
```
Comment (ckittel) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:284]:
```suggestion
        - Currently, Azure Synapse Analytics only supports disaster recovery for [dedicated SQL pools](/azure/synapse-analytics/sql-data-warehouse/backup-and-restore#geo-backups-and-disaster-recovery) and [doesn’t support it for Apache Spark pools](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-defender-for-key-vault-deploy-to-azure-synapse/ba-p/3201308)
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 5]
```diff
@@ -273,15 +273,15 @@ The following tables present a breakdown of each Azure service and component use
     - Contoso SKU selection: Computed Optimized, Small (4 cores)
     - DR Uplift options: N/A, Synapse resiliency is part of its SaaS offering
     - Notes
-        - Availability Zones are enabled by default for [Synapse Data Explorer](/azure/synapse-analytics/data-explorer/data-explorer-compare) where available
+        - Availability Zones are enabled by default for [Synapse Data Explorer](/azure/synapse-analytics/data-explorer/data-explorer-compare#:~:text=Enabled%20by%20default%20where%20Availability%20Zones%20are%20available) where available
```
Comment (ckittel) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:276]:
```suggestion
        - Availability Zones are enabled by default for [Synapse Data Explorer](/azure/synapse-analytics/data-explorer/data-explorer-compare) where available.
```

**Review by ckittel — APPROVED** [review]
I'll apply these edits, but approved otherwise.

**Comment by ckittel** [comment]
CACT #sign-off

#remove-label:"cact-review-in-progress"
#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Review by ckittel — APPROVED** [review]

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 6]
```diff
@@ -255,7 +255,7 @@ The following tables present a breakdown of each Azure service and component use
     - Contoso SKU selection: N/A
     - DR Uplift options: N/A
     - Notes
-        - As at Mar 2023, [Microsoft Purview doesn't support automated BCDR](/azure/purview/disaster-recovery#achieve-business-continuity-for-azure-purview). Until that support is added, the customer is responsible for all backup and restore activities.
+        - As at Dec 2023, [Microsoft Purview doesn't support automated BCDR](/azure/purview/disaster-recovery#achieve-business-continuity-for-azure-purview). Until that support is added, the customer is responsible for all backup and restore activities.
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:258]:
```suggestion
        - As of Dec 2023, [Microsoft Purview doesn't support automated BCDR](/azure/purview/disaster-recovery#achieve-business-continuity-for-azure-purview). Until that support is added, the customer is responsible for all backup and restore activities.
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 7]
```diff
@@ -293,23 +293,23 @@ The following tables present a breakdown of each Azure service and component use
         - Azure Synapse Analytics performs a [standard geo-backup](/azure/cloud-adoption-framework/migrate/azure-best-practices/analytics/azure-synapse#disaster-recovery) once per day to a paired data center. The RPO for a geo-restore is 24 hours
         - If Self-Hosted Data Pipelines are used, they'll remain the customers responsibility recovery from a disaster
 
-- **Azure Cognitive Services**
+- **Azure AI Services (formerly Cognitive Services)**
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:296]:
```suggestion
- **Azure AI services (formerly Cognitive Services)**
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 8]
```diff
@@ -293,23 +293,23 @@ The following tables present a breakdown of each Azure service and component use
...
     - Component Recovery Responsibility: Microsoft
     - Workload/Configuration Recovery Responsibility: Microsoft
     - Contoso SKU selection: Pay As You Go 
-    - DR Uplift options: N/A, the APIs for Cognitive Services are hosted by [Microsoft-managed data centers](/azure/cognitive-services/what-are-cognitive-services#regional-availability)
+    - DR Uplift options: N/A, the APIs for AI Services are hosted by [Microsoft-managed data centers](/azure/cognitive-services/what-are-cognitive-services#regional-availability)
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:300]:
```suggestion
    - DR Uplift options: N/A, the APIs for AI services are hosted by [Microsoft-managed data centers](/azure/cognitive-services/what-are-cognitive-services#regional-availability)
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 9]
```diff
@@ -293,23 +293,23 @@ The following tables present a breakdown of each Azure service and component use
...
     - Notes
-        -  If Cognitive Services has been deployed via customer deployed [Docker containers](/azure/cognitive-services/cognitive-services-container-support), recovery remains the responsibility of the customer
+        -  If AI Services has been deployed via customer deployed [Docker containers](/azure/cognitive-services/cognitive-services-container-support), recovery remains the responsibility of the customer
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:302]:
```suggestion
        -  If AI services has been deployed via customer deployed [Docker containers](/azure/cognitive-services/cognitive-services-container-support), recovery remains the responsibility of the customer
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md [hunk 10]
```diff
@@ -293,23 +293,23 @@ The following tables present a breakdown of each Azure service and component use
...
     - Component Recovery Responsibility: Microsoft
     - Workload/Configuration Recovery Responsibility: Microsoft
     - Contoso SKU selection: Standard S1
     - DR Uplift options: 
-        - Cognitive Search can be raised to an [HA design](/azure/search/search-performance-optimization#high-availability) by using replicas across [availability zones](/azure/search/search-performance-optimization#availability-zones) and regions
+        - AI Search can be raised to an [HA design](/azure/search/search-performance-optimization#high-availability) by using replicas across [availability zones](/azure/search/search-performance-optimization#availability-zones) and regions
         -  [Multiple services in separate regions](/azure/search/search-performance-optimization#multiple-services-in-separate-geographic-regions) can extend the resiliency further
     - Notes
-        - In Cognitive Search business continuity (and disaster recovery) is achieved through multiple search services.
+        - In AI Search business continuity (and disaster recovery) is achieved through multiple AI search services.
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md:312]:
```suggestion
        - In AI Search business continuity (and disaster recovery) is achieved through multiple AI Search services.
```

File: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-scenario-details-content.md [hunk 11]
```diff
@@ -205,7 +205,7 @@ The following sections present a breakdown of Contoso activity necessary across
     - Notes
         - A key feature of Stream Analytics is its ability to recover from [Node failure](/azure/stream-analytics/stream-analytics-concepts-checkpoint-replay#job-recovery-from-node-failure-including-os-upgrade)
 
-- **Azure Cognitive Services**
+- **Azure AI Services (formerly Cognitive Services)**
```
Comment (Court72) [inline docs/data-guide/disaster-recovery/dr-for-azure-data-platform-scenario-details-content.md:208]:
```suggestion
- **Azure AI services (formerly Cognitive Services)**
```

**Pull request merged**